curdleproofs/curdleproofs/test_curdleproofs.py

```diff
@@ -193,12 +197,14 @@ def test_py_arkworks_bls12381_scalar():
 
     assert CURVE_ORDER == 52435875175126190479447740508185965837690552500527637822603658699938581184513
 
-    # Why does Scalar does not support values over 2**128?
```

Comment:
Previously, the input to `__new__` was `u128`. We've updated it to be a `BigUint`.